include/roboptim/core/portability.hh

```diff
@@ -83,8 +83,8 @@
 
 // Required to avoid size_t resolution error with MSVC. Triggered by
 // the boost/tuple/tuple_io.hpp inclusion in roboptim/core/io.hh.
-#ifdef _WIN32
-# define BOOST_NO_STD_LOCALE
-#endif
+//#ifdef _WIN32
+//# define BOOST_NO_STD_LOCALE
+//#endif
```

Comment:
We should either remove this completely, or narrowly select the impacted MSVC versions.